conf/messages

```diff
@@ -3202,7 +3203,7 @@ annualExemptAmount.agent.error.tooSmall=Annual Exempt Amount must be 0 or more
 annualExemptAmount.trust.error.tooSmall=Annual Exempt Amount must be 0 or more
 annualExemptAmount.capacitor.error.tooSmall=Annual Exempt Amount must be 0 or more
 annualExemptAmount.personalRep.error.tooSmall=Annual Exempt Amount must be 0 or more
-annualExemptAmount.personalRepInPeriodOfAdmin.error.tooSmall=Annual Exempt Amount must be 0 or more
+annualExemptAmount.personalRepInPeriodOfAdmin.error.tooSmall=Annual Exempt Amount must be 0 or moreEx
```

Comment:
think you have added extra letters by mistake here

Reply:
fixed